lib/auth/saml.go

```diff
@@ -425,6 +524,12 @@ func (a *Server) validateSAMLResponse(samlResponse string) (*samlAuthResponse, e
 		Username: user.GetName(),
 	}
 
+	// In test flow skip signing and creating web sessions.
+	if request.SSOTestFlow {
+		diagCtx.info.Success = "test flow"
```

Comment:
`diagCtx.info.Success = "test flow"` looks a bit strange/random, is this "success" marker needed? If it is, why is it a string instead of a bool?

Reply:
I used a string for symmetry with `Error` and as a room for returning additional data. But I concur that this isn't really utilized that much and we could probably get away with just a bool. If we ever need to return extra data, we can simply add another field.